nuntium/models.py

```diff
@@ -151,6 +152,24 @@ class WriteItInstanceConfig(models.Model):
         help_text=_("Messages pushed to the api should \
             be confirmed automatically"), default=True)
 
+    custom_from_domain = models.CharField(max_length=512, null=True, blank=True, default=None)
```

Comment:
We could give it a go! But I was just a bit afraid that south would say something like "What do you want to do with backwards migrations?", so in order to avoid that sort of thing I just declare it. But it doesn't mean that it isn't the default by default.